### DISCUSSION TITLE
⭐ Xcode 16.2

### DISCUSSION THREAD

**Discussion by helinanever** [codemagic-ci-cd/codemagic-docs#2802] [Announcements]
**Xcode 16.2 Beta 1** is now available in Codemagic. This image comes with runtimes for iOS 18.2, tvOS 18.1, watchOS 11.1, and visionOS 2.1. To use it, set the Xcode version to `16.2` or `edge` in your build configuration.

See details about the build machine specs and preinstalled software in our [documentation](https://docs.codemagic.io/specs/versions-macos/).

**Comment by helinanever** [comment]
**Xcode 16.2 Beta 2** is now available on Codemagic build machines.

**Comment by mikhail-tokarev** [comment]
**Xcode 16.2 Beta 3 (16C5023f)** is now available on Codemagic build machines.

**Comment by mikhail-tokarev** [comment]
**Xcode 16.2 (16C5032a)** is now available on Codemagic build machines, along with **macOS Sequoia 15.2**
